system/jlib/jutil.cpp

```diff
@@ -56,6 +56,9 @@
 
 #include "portlist.h"
 
+using namespace std;
```

Comment:
picky: this isn't needed needed since your're fully qualifying the classes below.